src/sass_types/factory.cpp

```diff
@@ -45,26 +43,27 @@ namespace SassTypes
     }
   }
 
-  void Factory::initExports(Handle<Object> exports) {
-    Local<Object> types = NanNew<Object>();
-    exports->Set(NanNew("types"), types);
+  void Factory::initExports(v8::Local<v8::Object> exports) {
+    Nan::EscapableHandleScope scope;
```

Comment:
On Sun, 9 Aug 2015, Benjamin Byholm wrote:

> > -  void Factory::initExports(Handle<Object> exports) {
> > -    Local<Object> types = NanNew<Object>();
> > -    exports->Set(NanNew("types"), types);
> > +  void Factory::initExports(v8::Local<v8::Object> exports) {
> > +    Nan::EscapableHandleScope scope;
> 
> oh, and this could also use `NAN_MODULE_INIT` if it's being fed from `NODE_MODULE`.

Yes it is! Will rename the parameter.